src/index.js

```diff
@@ -0,0 +1,8 @@
+'use strict'
+
+module.exports = {
+  Stop: require('./circuit/hop'),
+  Dialer: require('./dialer'),
+  multicodec: require('./multicodec'),
+  tag: 'Circuit'
+}
```

Comment:
re `.getAddr` right now the only thing that circuit does a little different from other transports is that during instantiation it will check if there is `/p2p-circuit/<maddr>` address provided and if there isn't it will add a default `/p2p-circuit`, the reason for doing this is to enable circuit dialing/listening by default without having to repeat `/p2p-circuit` anywhere. I think this is OK to do, since `/p2p-circuit` is a perfectly valid circuit address that means that this peer is reachable over any circuit.